test/fixtures/url-tests-additional.js

```diff
@@ -0,0 +1,87 @@
+module.exports = [
```

Comment:
I can give it a try first :D. These test cases don't have `base` so is `urltestdata.json` the right place to go?